components/org.wso2.carbon.identity.api.server.tenant.management/org.wso2.carbon.identity.api.server.tenant.management.v1/src/main/resources/tenant-management.yaml

```diff
@@ -497,10 +493,9 @@ components:
       name: filter
       required: false
       description: >
-        Condition to filter the retrival of records. Supports 'sw', 'co', 'ew'
-        and 'eq' operations and also complex queries with 'and' operations. E.g.
-        /tenants?filter=domain+sw+"wso2". _<b>This
-        option is not yet supported.<b>_
+        Condition to filter the retrieval of records. Supports 'sw', 'co', 'ew' and 'eq' operations. Currently filtering
+        through <b>domainName</b> only supported.
```

Comment:
How about
Condition to filter the retrieval of records. Supports 'sw', 'co', 'ew' and 'eq' operations. Currently, filtering is supported only by the <b>domainName</b>.